internal/cmd/agent/deployer/monitor/updatestatus.go

```diff
@@ -81,7 +81,7 @@ func ShouldUpdateStatus(bd *fleet.BundleDeployment) bool {
 }
 
 func (m *Monitor) UpdateStatus(ctx context.Context, bd *fleet.BundleDeployment, resources *helmdeployer.Resources) (fleet.BundleDeploymentStatus, error) {
-	logger := log.FromContext(ctx).WithName("UpdateStatus")
+	logger := log.FromContext(ctx).WithName("update-status")
```

Comment:
We have a few other CamelCased names, eg. `HelmDeployer`, `deleteByRelease`, `getNamespacesForBundle`... Is there a reason not to convert those?

Reply:
Those must have escaped my grep.